librz/parse/p/parse_arm_pseudo.c

```diff
@@ -290,16 +290,20 @@ static bool subvar(RzParse *p, RzAnalysisFunction *f, RzAnalysisOp *op, char *da
 			if (!ripend) {
 				ripend = "]";
 			}
-			if (neg) {
-				repl_num -= rz_num_get(NULL, neg + 1);
-			} else {
-				repl_num += rz_num_get(NULL, rip);
+			const char * maybe_num = neg ? neg + 1 : rip;
+			while (*maybe_num == ' ') maybe_num ++;
```

Comment:
`rz_str_trim_head()` could be used instead probably

Reply:
changed already.